highfidelity/hifi
# Don't allow Web surfaces created by non-client scripts to access files

Fix for [16633](https://highfidelity.manuscript.com/f/cases/16633)

## Testing 

On a windows machine...

* Create a file on your local machine `C:\Users\bdavi\.ssh\id_rsa.pub` 
* Put something in the file
* Go into a domain where you have edit rights and create a cube
* Edit the properties of the cube and give it the script URL `https://sodium-pager-129219.appspot.com/browserSploit.js`
* As soon as the property is set and applied you should see an OverlayWebWindow pop up

On master builds the web window will have the contents of the file you created.  In this PR build, the web window should show an error page saying access to the file was _blocked_.

* Additionally, verify that existing _client_ scripts that rely on local file access still work
  * The edit.js entity properties editor should still render normally 
  * The Snapshot UI for showing a preview of the captured snapshot or gif works
